src/main/java/com/hazelcast/web/HazelcastInstanceLoader.java

```diff
@@ -100,32 +130,26 @@ private static Config getServerConfig(String mapName, URL configUrl, String sess
                 throw new ServletException(e);
             }
         }
-        if (sessionTTLConfig == null) {
-            sessionTTLConfig = SESSION_TTL_DEFAULT_SECONDS;
+
+        if (!isNullOrEmptyAfterTrim(instanceName)) {
```

Comment:
@mesutcelik @emre-aydin three of us agree that it's conceptually right approach. Yes, potentially it makes break existing applications after the upgrade. That's why this should be documented in a migration / release notes @Serdaro